src/objdump.py

```diff
@@ -72,7 +72,7 @@ def parse_relocated_line(line: str) -> Tuple[str, str, str]:
     return before, imm, after
 
 
-def simplify_objdump(input_lines: List[str]) -> List[str]:
+def simplify_objdump(input_lines: List[str], stack_differences: bool) -> List[str]:
```

Comment:
In general, bool parameters are very hard to understand what they do when you read the code at a call site. Prefer either passing in the entire Options object or a subobject thereof that deals only with scoring, or making this a keyword-only param:
```
def simplify_objdump(input_lines: List[str], *, stack_differences: bool) -> List[str]:
    ...
...
simplify_objdump(lines, stack_differences=True or False)
```